src/Native/System.Native/CMakeLists.txt

```diff
@@ -31,8 +31,17 @@ add_library(System.Native
     ${NATIVE_SOURCES}
 )
 
+
+add_library(System.Native.a
+    STATIC
+    ${NATIVE_SOURCES}
+)
+SET_TARGET_PROPERTIES(System.Native.a PROPERTIES PREFIX "")
```

Comment:
We should disable the prefix one level up - same way as it is done for shared libraries.